.gitignore

```
@@ -4,3 +4,4 @@
 # Build results
 bin/
 obj/
+/TestResults/

```

```suggestion
TestResults/
```
Does it work this way?

---

Looks like it. I'll change it in the updated PR so it is more consistent with the rest of the file.